src/Link.js

```diff
@@ -4,12 +4,17 @@ export function Link(props, children) {
   return function(state, actions) {
     var to = props.to
     var location = state.location || window.location
+    var onclick = props.onclick
     delete props.to
     delete props.location
 
     props.href = to
     props.onclick = function(e) {
+      if (onclick) {
+        onclick(e)
+      }
       if (
+        e.defaultPrevented ||
```

Comment:
@frenzzy I imagine this works because `e` is mutated when `e.preventDefault()` is run?

Reply:
Looks like it. 👍